Documentation/release-notes/4786.md

```diff
@@ -0,0 +1,13 @@
+#### Application behavior on device and emulator
+
+- [Developer Community 1045818][0]: Starting in Xamarin.Android 10.3,
+  errors similar to _Java.Lang.RuntimeException: 'Font asset not found
+  Fonts/fontello.ttf'_ could occur during an incremental build in
+  Visual Studio on Windows. There is a [known issue][1] with the
```

Comment:
I think the issue occurs when the app is running on device rather than during the build itself? So maybe a rewording of the first sentence could be:

```
[Developer Community 1045818][0]: Starting in Xamarin.Android 10.3 on Windows,
errors similar to _Java.Lang.RuntimeException: 'Font asset not found
Fonts/fontello.ttf'_ could cause apps to abort unexpectedly on device after they
were updated and redeployed from Visual Studio.
```